File: bot.lua
local discordia = require('discordia')
local client = discordia.Client()

client:on('ready', function()
	print('Logged in as '.. client.user.username)
end)

client:on('messageCreate', function(message)
	if message.content == '!ping' then
		message.channel:send('Pong!')
	end
end)

client:run('Bot NDQ2MTQ1Njg1MTkzMDk3MjI2.Dd1g5w.T7nvpqxsDql8D0is7TmNuNgdI5s')

File: bot.py
bot = commands.Bot(command_prefix='$')

@bot.event
async def on_ready():
    print('Logged in as')
    print(bot.user.name)
    print(bot.user.id)
    print('------')

@bot.command()
async def greet(ctx):
    await ctx.send(":smiley: :wave: Hello, there!")

bot.run('NDQ2MTQ1Njg1MTkzMDk3MjI2.Dd1zzg.nFyskkj31NdFYKCk-pm3HMrWad8')
